### ISSUE TITLE
Reflect.getMetadata is not a function

### ISSUE THREAD

**Issue by KingsDevelopment** [angular/angular#11720]
**I'm submitting a ...**  (check one with "x")
```
[ x ] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request => Please do not submit support request here, instead see https://github.com/angular/angular/blob/master/CONTRIBUTING.md#question
```

**Current behavior** 
After building, I open the application in the browser, and the only error I see is;
`Uncaught TypeError: Reflect.getMetadata is not a function`

This happened after upgrading rc7 to 2.0.0 (official)

**Expected behavior**
Run without errors.

**Reproduction of the problem**
Can't get it to happen on plunkr.

**What is the motivation / use case for changing the behavior?**
--

**Please tell us about your environment:**
Ubuntu server 14.04
npm 3.10.7
node 4.5.0

* **Angular version:** 2.0.0-rc.X
2.0.0

* **Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ] 
all
 
* **Language:** [all | TypeScript X.X | ES6/7 | ES5]
 TypeScript 2.0.2

* **Node (for AoT issues):** `node --version` =   
4.5.0

**Comment by vicb** [comment]
you must be missing the reflect-metadata package ?

Your question sounds like a support request.

Please use the issue tracker only for bugs and feature requests.

Use [gitter](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-got-a-question-or-problem) and [StackOverflow](http://stackoverflow.com/questions/tagged/angular) for support request.

**Comment by tinchou** [comment]
@KingsDevelopment it just happened to me, and the reason was I was using angular universal but missing at the beginning of my boot script:

```ts
import "angular2-universal-polyfills/node";
```

**Comment by KingsDevelopment** [comment]
@vicb I did try adding the `reflect-metadata` package, even though it does not state that it's needed:
https://angular.io/docs/ts/latest/quickstart.html > `package.json`

This happened after updating to `2.0.0`, without any changes, so I am wondering how I am to fault.

@tinchou  I am not using the universal.


My main.ts:

	import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
	import { enableProdMode } from '@angular/core';
	import { SuperBuddy, environment } from './app/';

	if (environment.production) {
	  enableProdMode();
	}

	platformBrowserDynamic().bootstrapModule(SuperBuddy);

**Comment by tinchou** [comment]
Look at Step 5 where they include the polyfills. Those should be in your
main.ts.

**Comment by btinoco** [comment]
It is happening to me too: "Uncaught TypeError: Reflect.getMetadata is not a function".  I am using angular 2.0.2, npm 3.10.5, node 4.7.7, MacOS 10.11.5

**Comment by pkozlowski-opensource** [comment]
@btinoco there is not much point in commenting on closed issues. If you need help / support please turn to one of the support channels: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#question

Regarding your issue you _might_ Reflect metadata polyffil in certain configurations, see: https://angular.io/docs/ts/latest/quickstart.html. But this is by no means specific to @ng-bootstrap (assuming that we talk about https://github.com/ng-bootstrap/ng-bootstrap). 

tl;dr; reach out for help on one of the support channels.

**Comment by Davy-F** [comment]
@btinoco Did you get this fixed? Also having this issue.

**Comment by rhrn** [comment]
@Davy-F find solution one here

https://stackoverflow.com/questions/40933065/testing-angular-2-service-with-mocha

**Comment by nuander** [comment]
Also having this issue.  I see an "answer" that says see step 5 to something that has no step 5.  I added polyfills.ts from another project and referenced it in my main.ts.  then made sure that the polyfill imports existed in my node_modules folder.  Still getting an error when trying to run app from VS Core

**Comment by phgachoud** [comment]
adding `import 'core-js/es7/reflect';` to my main.ts solved the problem for me
